src/main/scala/cromwell/binding/types/WdlFileType.scala

```diff
@@ -1,9 +1,5 @@
 package cromwell.binding.types
 
-import java.io.File
-
 case object WdlFileType extends WdlType {
-  def isCompatible(value: Any) = value.isInstanceOf[File]
-
   override def toString: String = "File"
```

Comment:
Shouldn't the WdlType implementations return things like `WdlFileType` (which is probably something we get for free)?